### PR TITLE
[PORT] Antag OOC

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

adds antag OOC

## Why It's Good For The Game

so admins can yell at the antags without IC in OOC.

image

ORIGINAL PR: https://github.com/OracleStation/OracleStation/pull/73

## Changelog
:cl:
add: Added antag OOC
/:cl:

### CODE REVIEW

**Comment by Archanial** [comment]
admins can just bwoink antags?

**Comment by ghost** [comment]
> admins can just bwoink antags?

team antags, or generally saying something to the antags of the current round, this isn't just for rule breakings.

**Comment by PowerfulBacon** [comment]
For team antags there is a button to send a message to the whole team

File: code/modules/admin/admin.dm [hunk 1]
```diff
@@ -736,6 +736,28 @@
...
+	var/list/to_hear = list()
+	for(var/client/C in GLOB.clients)
+		if(C in GLOB.admins)
+			to_hear += C
+			continue
+		if(C.mob && is_special_character(C.mob))
+			to_hear += C
+
+	for(var/C in to_hear)
+		to_chat(C, "<span class='aooc'>AOOC: [usr.key]: [message]</span>")
```
Comment (PowerfulBacon) [inline code/modules/admin/admin.dm:758]: Why is it done like this? Why not just have the to_chat in the first loop

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]

**Comment by qwertyquerty** [comment]
This is such a bad concept in general imo

**Comment by qwertyquerty** [comment]
@PowerfulBacon @zeskorion and I all agreed to just close this

**Pull request closed without merging**